python/pyspark/pandas/tests/test_dataframe_spark_io.py

```diff
@@ -96,11 +97,18 @@ def test_parquet_read_with_pandas_metadata(self):
             self.assert_eq(ps.read_parquet(path2, pandas_metadata=True), expected2)
 
             expected3 = expected2.set_index("index", append=True)
+            # There is a bug in `to_parquet` from pandas 1.5.0 when writing MultiIndex.
+            # See https://github.com/pandas-dev/pandas/issues/48848 for the reported issue.
+            if LooseVersion(pd.__version__) >= LooseVersion("1.5.0"):
```

Comment:
```suggestion
            if LooseVersion(pd.__version__) == LooseVersion("1.5.0"):
```

maybe .. since it's going to be fixed in 1.5.1 presumably.